index.js

```diff
@@ -52,7 +52,8 @@ const ignored = [
   },
   {
     property: "display",
-    value: "/^table-.*$/",
+    value:
+      "/^table-(row|row-group|column|column-group|header-group|footer-group|cell)$/",
```

Comment:
> margin has no effect on this element since it is has a table-cell display type
margin has no effect on this element since it is has a table-column display type
margin has no effect on this element since it is has a table-column-group display type
margin has no effect on this element since it is has a table-footer-group display type
margin has no effect on this element since it is has a table-header-group display type
margin has no effect on this element since it is has a table-row display type
margin has no effect on this element since it is has a table-row-group display type

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#margin-and-padding-properties-on-tables

---
> margin: applies to all elements, except elements with table display types other than table-caption, table and inline-table.

https://developer.mozilla.org/en-US/docs/Web/CSS/margin